src/NumericalAlgorithms/Interpolation/InterpolationTargetReceiveVars.hpp

```diff
@@ -136,16 +136,22 @@ void fill_invalid_points(const gsl::not_null<db::DataBox<DbTags>*> box,
   }
 }
 
-/// Calls the callback function, tells interpolators to clean up the current
-/// temporal_id, and then if there are more temporal_ids to be interpolated,
-/// starts the next one.
+/// Prepares for calling the callback function on an InterpolationTarget,
+/// calls the function, and returns true if the callback is done and
+/// false if the callback is not done (e.g. if the callback is an
+/// apparent horizon search and it needs another iteration).
+///
+/// call_callback is called by an Action of InterpolationTarget.
+///
+/// Currently two Actions call call_callback:
+///   InterpolationTargetReceiveVars (called by Interpolator ParallelComponent)
+///   InterpolationTargetVarsFromElement (called by DgElementArray)
```

Comment:
I think this will display better in doxygen if the list is formatted as either
```
- foo
- bar
```
or
```
1. foo
2. bar
```

(Same comment applies to other function documentations below)

Reply:
:+1: